massa-final-state/src/final_state.rs

```diff
@@ -635,7 +635,16 @@ impl FinalState {
                     ));
                 }
             } else if serialized_key.starts_with(MIP_STORE_PREFIX.as_bytes()) {
-                // TODO: check MIP_STORE_PREFIX
+                if !self
+                    .mip_store
+                    .is_key_value_valid(&serialized_key, &serialized_value)
+                {
+                    warn!("Wrong key/value for MIP Store");
+                    return Err(anyhow!(
+                        "Wrong key/value for MIP Store serialized_key: {:?}, serialized_value: {:?}", 
+                        serialized_key, serialized_value
```

Comment:
I don't know how big `serialized_key` and `serialized_value` can be.
If they can be bigger than a "limit", maybe we want to use `.get(0..limit)` to return only their beginning

Reply:
fine